### PR TITLE
Add documentation for LogarithmicScaleDown featuregate

### PR DESCRIPTION
**Pull request by MinpengJin**
ref: https://github.com/kubernetes/enhancements/issues/2185

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates/logarithmic-scale-down.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ stages:
   - stage: beta
     defaultValue: true
     fromVersion: "1.22"
+  - stage: stable
+    defaultValue: true
+    fromVersion: "1.31"
```
Comment (dipesh-rawat) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/logarithmic-scale-down.md:18]: Adding `toVersion` to the previous stage would enhance consistency with other feature gate files.

```suggestion
    toVersion: "1.30"
  - stage: stable
    defaultValue: true
    fromVersion: "1.31"
```

Comment (MinpengJin) [reply content/en/docs/reference/command-line-tools-reference/feature-gates/logarithmic-scale-down.md:19]: done

File: content/en/docs/concepts/workloads/controllers/replicaset.md [hunk 2]
```diff
@@ -339,8 +339,7 @@ prioritize scaling down pods based on the following general algorithm:
    the pod with the lower value will come first.
 1. Pods on nodes with more replicas come before pods on nodes with fewer replicas.
 1. If the pods' creation times differ, the pod that was created more recently
-   comes before the older pod (the creation times are bucketed on an integer log scale
-   when the `LogarithmicScaleDown` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled)
+   comes before the older pod (the creation times are bucketed on an integer log scale)
```
[the remark this answers is not in the document]
Comment (MinpengJin) [reply content/en/docs/concepts/workloads/controllers/replicaset.md]: done

**Comment by windsonsea** [comment]
Thanks.
/lgtm

**Pull request merged**